CMakeLists.txt

```diff
@@ -78,44 +96,125 @@ add_library(lcf
+	src/generated/lsd_savepicture_flags.h
 	src/generated/lsd_savescreen.cpp
 	src/generated/lsd_savesystem.cpp
 	src/generated/lsd_savetarget.cpp
 	src/generated/lsd_savetitle.cpp
 	src/generated/lsd_savevehiclelocation.cpp
+	src/generated/rpg_actor.h
+	src/generated/rpg_animation.h
+	src/generated/rpg_animationcelldata.h
+	src/generated/rpg_animationframe.h
+	src/generated/rpg_animationtiming.h
+	src/generated/rpg_attribute.h
+	src/generated/rpg_battlecommand.h
+	src/generated/rpg_battlecommands.h
+	src/generated/rpg_battleranimation.h
+	src/generated/rpg_battleranimationdata.h
+	src/generated/rpg_battleranimationextension.h
 	src/generated/rpg_chipset.cpp
+	src/generated/rpg_chipset.h
+	src/generated/rpg_class.h
+	src/generated/rpg_commonevent.h
+	src/generated/rpg_database.h
+	src/generated/rpg_encounter.h
+	src/generated/rpg_enemy.h
+	src/generated/rpg_enemyaction.h
 	src/generated/rpg_enums.cpp
+	src/generated/rpg_equipment.h
+	src/generated/rpg_event.h
+	src/generated/rpg_eventcommand.h
+	src/generated/rpg_eventpage.h
+	src/generated/rpg_eventpagecondition.h
+	src/generated/rpg_item.h
+	src/generated/rpg_itemanimation.h
+	src/generated/rpg_learning.h
+	src/generated/rpg_map.h
+	src/generated/rpg_mapinfo.h
+	src/generated/rpg_movecommand.h
+	src/generated/rpg_moveroute.h
+	src/generated/rpg_music.h
+	src/generated/rpg_parameters.h
+	src/generated/rpg_rect.h
+	src/generated/rpg_save.h
+	src/generated/rpg_saveactor.h
+	src/generated/rpg_savecommonevent.h
+	src/generated/rpg_saveeventcommands.h
+	src/generated/rpg_saveeventdata.h
+	src/generated/rpg_saveinventory.h
+	src/generated/rpg_savemapevent.h
+	src/generated/rpg_savemapeventbase.h
+	src/generated/rpg_savemapinfo.h
+	src/generated/rpg_savepanorama.h
 	src/generated/rpg_savepartylocation.cpp
+	src/generated/rpg_savepartylocation.h
+	src/generated/rpg_savepicture.h
+	src/generated/rpg_savescreen.h
+	src/generated/rpg_savesystem.h
+	src/generated/rpg_savetarget.h
+	src/generated/rpg_savetitle.h
+	src/generated/rpg_savevehiclelocation.h
+	src/generated/rpg_skill.h
+	src/generated/rpg_sound.h
+	src/generated/rpg_start.h
 	src/generated/rpg_state.cpp
+	src/generated/rpg_state.h
+	src/generated/rpg_switch.h
 	src/generated/rpg_system.cpp
+	src/generated/rpg_system.h
+	src/generated/rpg_terms.h
+	src/generated/rpg_terrain.h
+	src/generated/rpg_testbattler.h
+	src/generated/rpg_treemap.h
+	src/generated/rpg_troop.h
+	src/generated/rpg_troopmember.h
+	src/generated/rpg_trooppage.h
+	src/generated/rpg_trooppagecondition.h
+	src/generated/rpg_variable.h
 )
 
+# IDE source grouping
+foreach(SG LDB LMT LMU LSD RPG)
+	string(TOLOWER ${SG} LSG)
+	source_group("Source Files\\generated\\${SG}" REGULAR_EXPRESSION "generated/${LSG}_.*\\.cpp")
+	source_group("Header Files\\generated\\${SG}" REGULAR_EXPRESSION "generated/${LSG}_.*\\.h")
+	source_group("Source Files\\${SG}" REGULAR_EXPRESSION "${LSG}_.*\\.cpp")
+	source_group("Header Files\\${SG}" REGULAR_EXPRESSION "${LSG}_.*\\.h")
```

Comment:
Can you tell me any IDE (on Linux) where I can see the result?

Reply:
e.g. Eclipse:
`$ cmake . -DCMAKE_BUILD_TYPE=Release -G "Eclipse CDT4 - Ninja" -DCMAKE_ECLIPSE_VERSION=4.9`

![bloat](https://p.f4ke.de/8iUtxXchpq.png)